ndcube/global_coords/global_coords.py

```diff
@@ -33,19 +100,20 @@ def _all_coords(self):
         if self._ndcube is None:
             return self._internal_coords
 
+        all_coords = {**self._internal_coords}
+
         if hasattr(self._ndcube.wcs.low_level_wcs, "dropped_world_dimensions"):
-            dropped_world = self._ndcube.wcs.low_level_wcs.dropped_world_dimensions
-            wcs_dropped = {}
-            if dropped_world != {}:
-                for i in range(len(dropped_world["value"])):
-                    name = (dropped_world["world_axis_names"][i] or
-                            dropped_world["world_axis_physical_types"][i])
-                    val = dropped_world["value"][i] * u.Unit(dropped_world["world_axis_units"][0])
-                    physical_type = dropped_world["world_axis_physical_types"][i]
-                    wcs_dropped[name] = (physical_type, val)
-                return {**wcs_dropped, **self._internal_coords}
-
-        return self._internal_coords
+            dropped_world = copy.deepcopy(self._ndcube.wcs.low_level_wcs.dropped_world_dimensions)
+            if dropped_world:
+                wcs_dropped = self._convert_dropped_to_internal(dropped_world)
+                all_coords.update(wcs_dropped)
+
+        # TODO: Implement dropped_world_dimensions on ExtraCoords
```

Comment:
If not already done, raise issue (or put comment on existing issue) and add to 2.0 project.

Reply:
It is the remainder of #349